domain-locker/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: domain-locker_app_1
+      APP_PORT: 3000
+
+  app:
+    image: lissy93/domain-locker:0.0.6@sha256:bd8b494b2a8b02e9fb612859f64c336e6eb1b21745199253b130705bd4ac8853
+    user: "1000:1000"
+    restart: on-failure
+    environment:
+      DL_ENV_TYPE: ${DL_ENV_TYPE:-selfHosted}
+      DL_PG_HOST: ${DL_PG_HOST:-postgres}
+      DL_PG_PORT: ${DL_PG_PORT:-5432}
+      DL_PG_USER: ${DL_PG_USER:-postgres}
+      DL_PG_PASSWORD: ${DL_PG_PASSWORD:-changeme2420}
+      DL_PG_NAME: ${DL_PG_NAME:-domain_locker}
+    depends_on:
+      - postgres
+
+  postgres:
+    image: postgres:15-alpine@sha256:ef9d1517df69c4d27dbb9ddcec14f431a2442628603f4e9daa429b92ae6c3cd1

```

Comment:
Maybe we could try a more up-to-date version of postgres here like 17.4, just so we don't have to update this soonish.